documentation/modules/ROOT/pages/configuration/serdes.adoc

```diff
@@ -102,4 +102,8 @@ The deserializer behaviour is driven by the `from.field` configuration option an
 |`from.field`
 |`N/A`
 |Empty if a message with full envelope should be deserialized, `before`/`after` if only data values before or after the change are required.
+
+|`unknown.properties.ignored`
+|`false`
```

Comment:
The doc update makes me wonder: do we need to update the Antora docs so that the 1.0 branch gets published from the 1.0 tag instead of master?

Reply:
Most definitely, I'll push a fix for that shortly.

Reply:
Thanks!